src/instance.jl

```diff
@@ -283,32 +284,36 @@ end
 # Expr(:., MOI, $s) would be Expr(:., MOI, EqualTo)
 # Expr(:., MOI, :($s)) would be Expr(:., MOI, :EqualTo)
 # Expr(:., MOI, :($(QuoteNode(s)))) is Expr(:., MOI, :(:EqualTo)) <- what we want
-_mod(m, s::Symbol) = Expr(:., m, :($(QuoteNode(s))))
+
+# _mod(MOI, :Zeros) -> :(MOI.Zeros)
```

Comment:
Something is off with the code highlighting and `# _mod`, may need to remove the underscore.

Reply:
Seem to be due to https://github.com/atom/language-gfm/issues/44